src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -62,12 +62,10 @@ class HoldRequest extends React.Component {
     this.checkEligibility = this.checkEligibility.bind(this);
     this.conditionallyRedirect = this.conditionallyRedirect.bind(this);
     this.isLoading = this.isLoading.bind(this);
-    this.requireUser = this.requireUser.bind(this);
   }
 
 
   componentDidMount() {
-    this.requireUser();
```

Comment:
So, not doing the client-side check at all? Is there a reason not to? On slower connections, seems a shame to wait for the server to tell us we're not logged in, when we usually know that client-side

Reply:
Good point about it being a faster redirect in this case. I've added it back in, but theoretically it should be redundant.